src/main/resources/artifact-ignores.properties

```diff
@@ -118,3 +118,16 @@ scm-api-2.0
 # Suppress Scala Hello World Plugin (HOSTING-235)
 hello-world-scala
 
+
+# Suppress potentially harmful upgrades related to SCM API (JENKINS-41121)
+# Erring on the side of extreme caution until the fallout is resolved here
+scm-api:2.0.0
```

Comment:
While this syntax looks sensible, the only accepted syntax for this file is:
* `<artifact-id>`
* `<artifact-id>-<version>`

i.e. colons -> hyphens